src/__test__/hyperfine.test.ts

```diff
@@ -1,7 +1,7 @@
-import assert from 'node:assert';
+import { equal } from 'node:assert';
 import { describe, it } from 'node:test';
 
-import { Hyperfine } from '../index.js';
+import { Hyperfine } from '../index';
```

Comment:
```suggestion
import { Hyperfine } from '../index.js';
```

This breaks the typechecker

> TS2835: Relative import paths need explicit file extensions in EcmaScript imports when '--moduleResolution' is 'node16' or 'nodenext'.